docs/FAQ.md

```diff
@@ -0,0 +1,58 @@
+# Frequently asked questions
+
+### Why does my branch not move to the new commit after `jj new/commit`?
+
+Jujutsu's branches are akin to Git Tags, they always point to a single commit.
+To move them, use `jj branch set`. 
+
+### Where is my commit, why is it not visible in `jj log`? 
+
+If you don't find it with `jj log -r 'all()'`, it may be time to set the config
+`revsets.log` to something more matching.
+
+See [revsets], [templates] and [config] for guidance.
+
+### Is there something like `git rebase`? 
+
+For now its recommended to rebase the commit individually with 
+`jj rebase -s $changeid -d $new-parent`.
+
+In the future there may be a `histedit` command, which alleviates that.
+
+
+### How do I partially add changes from a file? 
+
+With Jujutsu you never partially add hunks, as your always amending the 
+`@` commit. You need to split it, as the `@` may already be complete.
```

Comment:
I think you should delete the "as the `@` may already be complete" and combine the paragraphs. 

What did you mean by this clause, by the way? If it's important, we can figure out another way to say it.

Reply:
This answer is based on ocharles [reply](https://discord.com/channels/968932220549103686/969829516539228222/1135593961805848687) on how to think about split in Discord. I think the phrasing should be improved.